packages/connectivity/src/scp-cf/destination/destination.ts

```diff
@@ -469,7 +432,7 @@ export function noDestinationErrorMessage(
 /**
  * Type that is either a {@link Destination} or (XOR) {@link DestinationFetchOptions & DestinationForServiceBindingOptions}.
  */
-export type DestinationOrFetchOptions = Xor<
-  Destination,
+export type DestinationOrFetchOptions<T extends Destination> = Xor<
```

Comment:
[pp] No strong opinion, but I have 2 thoughts:

1. `DestinationOrFetchOptions<HttpDestination>` is longer than `HttpDestinationOrFetchOptions`, therefore I am not sure whether the generic is better than just two types.
2. Could we find a better (shorter) name than `Destination....<Destination>`? My ideas do not convince me: `FetchOptionsOr<Destination>`,

Reply:
In principle I like the idea to use generic as part of the name as in `FetchOptionsOr<Destination>` however this is not used anywhere else.

Reply:
I use two types now